Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/compositeeditor/EditorModel.java

```diff
@@ -322,8 +322,7 @@ public void setComponentName(int rowIndex, String name)
 	public void cycleDataType(CycleGroup cycleGroup);
 
 	/**
-	 * 
-	 * @param parent
+	 *
 	 * @throws UsrException
 	 */
```

Comment:
Yeah empty javadoc doesn't really help, but I tried not to make too many changes, but this param for sure should be deleted, so I only removed this one, because it's a wrong reference